cmd/daprd/options/options.go

```diff
@@ -168,7 +168,7 @@ func New(origArgs []string) (*Options, error) {
 	// --placement-host-address is a legacy (but not deprecated) flag that is translated to the actors-service flag
 	var placementServiceHostAddr string
 	fs.StringVar(&placementServiceHostAddr, "placement-host-address", "", "Addresses for Dapr Actor Placement servers (overrides actors-service)")
-	fs.StringSliceVar(&opts.SchedulerAddress, "scheduler-host-address", nil, "Addresses of the Scheduler service instance(s), as comma separated host:port pairs")
+	fs.StringSliceVar(&opts.SchedulerAddress, "scheduler-host-address", []string{"127.0.0.1:50006"}, "Addresses of the Scheduler service instance(s), as comma separated host:port pairs")
```

Comment:
Does this not mean a bare daprd run without any other control plane service would error? It's currently always been the case that no flag was needed to be set to empty or nil on a bare daprd to run successfully. Strictly only --app-id iirc

Reply:
Btw, running a bare daprd binary (or container) with no other services is something we definitely want to (continue to) support

Reply:
Yes, this code should not make Dapr exit. Currently running a bare `daprd` with this line is entering a retry loop on the scheduler connection which isn't ideal. This behavior should be modified

Reply:
If the sidecar does not connect to the scheduler, it should not continue. It is unhealthy condition.